environments/core/applicationsets/kube-prometheus-stack-applicationset.yaml

```
@@ -17,7 +17,7 @@ spec:
         targetRevision: HEAD
       - cluster: devsecops-testing
         cluster-name: devsecops-testing
-        targetRevision: HEAD
+        targetRevision: chore/devsecops_kube_upgrade
```

```suggestion
        targetRevision: chore/devsecops_kube_prometheus_stack_appset_upgrade
````

should this one not changed to `chore/devsecops_kube_prometheus_stack_appset_upgrade` ?

---

chore/devsecops_kube_prometheus_stack_appset_upgrade branch is for ApplicationSet change but chore/devsecops_kube_upgrade is for actual version change of the kube prometheus stack (Chart.yaml will be modified there)

---

kay no problem, but `chore/devsecops_kube_upgrade` seems not available in the repo so was confused. so i guess need a little push 😉 that argo will be have a target revision to checkout for devsecops cluster